### ISSUE TITLE
upload failure: block length incorrect

### ISSUE THREAD

**Issue by AssRap3r** [topac/rmega#37]
when a filename has special characters (eg: accents, japanese) the upload will fail with the following error (obtained with --debug)

```
/var/lib/gems/2.3.0/gems/rmega-0.2.6/lib/rmega/crypto/aes_cbc.rb:13:in `final': data not multiple of block length (OpenSSL::Cipher::CipherError)
        from /var/lib/gems/2.3.0/gems/rmega-0.2.6/lib/rmega/crypto/aes_cbc.rb:13:in `aes_cbc_encrypt'
        from /var/lib/gems/2.3.0/gems/rmega-0.2.6/lib/rmega/nodes/uploadable.rb:73:in `upload'
        from /var/lib/gems/2.3.0/gems/rmega-0.2.6/bin/rmega-up:44:in `block in <top (required)>'
        from /var/lib/gems/2.3.0/gems/rmega-0.2.6/lib/rmega/cli.rb:117:in `cli_rescue'
        from /var/lib/gems/2.3.0/gems/rmega-0.2.6/bin/rmega-up:32:in `<top (required)>'
        from /usr/local/bin/rmega-up:23:in `load'
        from /usr/local/bin/rmega-up:23:in `<main>'
```
example:
rmega-up 10GB.párt00112.rar -t8 -u mysecretemail
10GB.párt00112.rar
[Upload] 100.1 MB of 100.1 MB. Completed in 7.28 sec.
ERROR: data not multiple of block length 
(file does not appear in MEGA account)

rmega-up 10GB.part001.rar -t8 -u mysecretemail
10GB.part001.rar
[Upload] 100.1 MB of 100.1 MB. Completed in 5.19 sec.
(file appears in MEGA account)

Ubuntu 16.04.6 LTS
Linux 2.6.32-042stab140.1 #1 SMP Thu Aug 15 13:32:22 MSK 2019 x86_64 x86_64 x86_64 GNU/Linux

bigly sad!

**Comment by topac** [comment]
Hi there, it's true! I tried uploading a file with a in it and it fails BUT with another error:
```

**Comment by AssRap3r** [comment]
updooted ruby to 2.7.1 and latest nodej v12.18.1

```
Traceback (most recent call last):
        9: from /home/sneedbox/.rvm/gems/ruby-2.7.1/bin/ruby_executable_hooks:24:in `<main>'
        8: from /home/sneedbox/.rvm/gems/ruby-2.7.1/bin/ruby_executable_hooks:24:in `eval'
        7: from /home/sneedbox/.rvm/gems/ruby-2.7.1/bin/rmega-up:23:in `<main>'
        6: from /home/sneedbox/.rvm/gems/ruby-2.7.1/bin/rmega-up:23:in `load'
        5: from /home/sneedbox/.rvm/gems/ruby-2.7.1/gems/rmega-0.2.6/bin/rmega-up:32:in `<top (required)>'
        4: from /home/sneedbox/.rvm/gems/ruby-2.7.1/gems/rmega-0.2.6/lib/rmega/cli.rb:117:in `cli_rescue'
        3: from /home/sneedbox/.rvm/gems/ruby-2.7.1/gems/rmega-0.2.6/bin/rmega-up:44:in `block in <top (required)>'
        2: from /home/sneedbox/.rvm/gems/ruby-2.7.1/gems/rmega-0.2.6/lib/rmega/nodes/uploadable.rb:73:in `upload'
        1: from /home/sneedbox/.rvm/gems/ruby-2.7.1/gems/rmega-0.2.6/lib/rmega/crypto/aes_cbc.rb:13:in `aes_cbc_encrypt'
/home/sneedbox/.rvm/gems/ruby-2.7.1/gems/rmega-0.2.6/lib/rmega/crypto/aes_cbc.rb:13:in `final': data not multiple of block length (OpenSSL::Cipher::CipherError)

```

**Comment by topac** [comment]
can you try with 1 thread? use the flag `-t 1`

**Comment by topac** [comment]
fixed in rmega 0.2.7
